pkg/support/dump.go

```diff
@@ -55,6 +58,17 @@ type ControllerDumper interface {
 	// DumpNetworkPolicyResources should create files that contains networkpolicy
 	// resources on the controller Pod under the base dir.
 	DumpNetworkPolicyResources(basedir string) error
+	// DumpHeapPprof should create a pprof file of the heap usage of the controller.
```

Comment:
It is for both controller and agent?

Reply:
Yes, it is.

Reply:
I mean would you update the comments to add agent.

Reply:
Yeah, the comment for `AgentDumper` is on :45.

Reply:
Ok. Did not see it.